pytx/pytx/common.py

```diff
@@ -70,7 +70,7 @@ def __setattr__(self, name, value):
 
         object.__setattr__(self, name, value)
         if name == c.ID:
-            self._DETAILS = t.URL + value + '/'
+            self._DETAILS = t.URL + t.VERSION + value + '/'
```

Comment:
Er... won't this result in something like:

    graph.facebook.com/2.499999999999/

Shouldn't there be a slash between the version and the value? Or am I reading this wrong?